default solidjs to use style-tag styling (#1494)
BuilderIO/mitosis: .changeset/many-doors-scream.md

---
"@builder.io/mitosis": patch
---

> Update Solid generator to default to `style-tag` styling instead of `solid-styled-components`.

Reviewer: ```suggestion
- Fix: [Solid] change style default to `style-tag` instead of `solid-styled-components`.
- Fix: [Solid] remove `jsx` attribute from `<style>` tags in `style-tag`.
```

> Default solidjs to use style-tag styling instead of solid-styled-components because of issues people were having with solid-styled-components

Reviewer: ```suggestion
Update Solid generator to default to `style-tag` styling instead of `solid-styled-components`.
```
